pajbot/modules/actionchecker.py

```diff
@@ -79,13 +79,23 @@ class ActionCheckerModule(BaseModule):
             default=500,
             constraints={"min_value": 100, "max_value": 1000},
         ),
+        ModuleSetting(
+            key="disable_warnings",
+            label="Disable warning timeouts",
+            type="boolean",
+            required=True,
+            default=False,
+        ),
     ]
 
     def delete_or_timeout(self, user, msg_id, reason):
         if self.settings["moderation_action"] == "Delete":
             self.bot.delete_message(msg_id)
         elif self.settings["moderation_action"] == "Timeout":
-            self.bot.timeout(user, self.settings["timeout_length"], reason, once=True)
+            if self.settings["disable_warnings"] is True:
+                self.bot.timeout(user, self.settings["timeout_length"], reason)
+            else:
+                self.bot.timeout_warn(user, self.settings["timeout_length"], reason)
```

Comment:
We've lost the `once=True` flag here, readd it please :)

Reply:
```suggestion
            if self.settings["disable_warnings"] is True:
                self.bot.timeout(user, self.settings["timeout_length"], reason, once=True)
            else:
                self.bot.timeout_warn(user, self.settings["timeout_length"], reason, once=True)
```